frameworks/helloworld/tests/scale/test_scale.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+import logging
+import sdk_install
+import sdk_security
+import sdk_dcos
+from tests import config
+
+log = logging.getLogger(__name__)
+
+ACCOUNTS = {}
```

Comment:
Do we need this global?

Reply:
This is not needed, was going to multi thread the creation of service accounts like we do in jenkins but for now until we run this test and get more info there is no need to do this. I like the suggestions you outlined